### ISSUE TITLE
add implementation of SMS-EMOA

### ISSUE THREAD

**Issue by jakobbossek** [jakobbossek/ecr#82]
Since this is a quite easy to implement and powerful EMOA, this one should be the first to implement.

**Comment by berndbischl** [comment]
Olafs version should still be only as a starting point

**Comment by jakobbossek** [comment]
I do not know whether it might be helpful, but where can I find Olafs sms-emoa?

**Comment by berndbischl** [comment]
http://git.p-value.net/p/emoa.git/tree/examples

**Comment by jakobbossek** [comment]
Thanks.

**Comment by jakobbossek** [comment]
It works!
Did a short test on the ZDT1 function with a slightly simplified version of the SMS-EMOA which considers only the dominance number as the survival selection criterion and not the hypervolume, since the latter is not implemented/integrated yet. But in principle EMOAs can now be implemented.
I pushed that to a separate branch since I had to bypass some single-objective specific stuff. Going to continiue work on this next week.
